packages/genesys-spark-components/src/components/beta/gux-avatar-group/gux-avatar-group.tsx

```diff
@@ -0,0 +1,203 @@
+import {
+  Component,
+  h,
+  JSX,
+  Element,
+  State,
+  Prop
+  // writeTask
+} from '@stencil/core';
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxAvatarAccent, GuxAvatarSize } from '../gux-avatar/gux-avatar.types';
+
+interface ProcessedAvatar {
+  focusableAvatar: HTMLGuxAvatarFocusableBetaElement;
+  avatar: HTMLGuxAvatarBetaElement;
+  button?: HTMLButtonElement;
+  link?: HTMLAnchorElement;
+  img?: HTMLImageElement;
+  name: string;
+  accent: GuxAvatarAccent;
+  isOverflow: boolean;
+}
+
+/**
+ * @slot - Some gux-avatars
```

Comment:
Does this slot still exist or no longer in use ? I was just wondering as I did not know it was possible to have a component that has no slot since it technically slots gux-avatar-focusable-beta.